src/state_db.rs

```diff
@@ -854,7 +853,7 @@ impl StateDB {
     pub fn js_commit(mut ctx: FunctionContext) -> JsResult<JsUndefined> {
         let writer = ctx.argument::<state_writer::SendableStateWriter>(0)?;
 
-        let key_length = ctx.argument::<JsNumber>(1)?.value(&mut ctx).into();
+        let block_height = ctx.argument::<JsNumber>(1)?.value(&mut ctx).into();
```

Comment:
here maybe we could name it `version` to be consistent with rest of the code

Reply:
Done.